lib/rubocop/cop/chef/redundant/ohai_attribute_to_string.rb

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright:: 2020, Chef Software, Inc.
+# Author:: Tim Smith (<tsmith@chef.io>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module RuboCop
+  module Cop
+    module Chef
+      module ChefRedundantCode
+        # Many Ohai node attributes are already strings and don't need to be cast to strings again
+        #
+        # @example
+        #
+        #   # bad
+        #   node['platform'].to_s
+        #   node['platform_family'].to_s
+        #   node['platform_version'].to_s
+        #   node['fqdn'].to_s
+        #   node['hostname'].to_s
+        #   node['os'].to_s
+        #   node['name'].to_s
+        #
+        #   # good
+        #   node['platform']
+        #   node['platform_family']
+        #   node['platform_version']
+        #   node['fqdn']
+        #   node['hostname']
+        #   node['os']
+        #   node['name']
+        #
+        class OhaiAttributeToString < Cop
+          MSG = "Many Ohai node attributes are already strings and don't need to be cast to strings again".freeze
```

Comment:
This error message kind of sounds like it's warning about using `.to_s` on any attribute. Since this is looking for a specific list of things and only warning for them, would something like this be more appropriate?

```suggestion
          MSG = "This Ohai node attribute is already a string and doesn't need to be converted".freeze
```